pwa/src/templates/templateParts/organizationsForm/OrganizationFormTemplate.module.css

```diff
@@ -58,3 +58,21 @@
     --gateway-ui-color-dark-red
   );
 }
+
+.tabContainer {
+  background: var(--gateway-ui-color-white);
+  margin-block-start: var(--gateway-ui-size-2xl);
+  padding-block: var(--gateway-ui-size-md);
+  padding-inline: var(--gateway-ui-size-md);
+}
+
+.tabPanel {
+  padding-inline-start: var(--gateway-ui-size-lg) !important;
+  padding-inline-end: var(--gateway-ui-size-lg) !important;
+  padding-block-start: var(--gateway-ui-size-lg) !important;
+  padding-block-end: var(--gateway-ui-size-lg) !important;
+}
+
+.tab {
+  max-width: unset !important;
+}
```

Comment:
Out of scope for this pull request, I think, but: we should probably move this to a more generic place.

Reply:
I agree